src/mlpack/core/data/split_data.hpp

```diff
@@ -44,6 +44,8 @@ namespace data {
  * @param trainLabel Vector to store training labels into.
  * @param testLabel Vector to store test labels into.
  * @param testRatio Percentage of dataset to use for test set (between 0 and 1).
+ * @param shuffleData If true, the sample order is shuffled; otherwise, each
+ *       sample is visited in linear order. (Default true).
```

Comment:
```suggestion
 *       sample is visited in linear order. (Default true.)
```

This is super pedantic, sorry about that. :)

Reply:
Did